web/src/locales/sv.json

```diff
@@ -221,7 +221,7 @@
     }
   },
   "header": {
-    "get-data": "Hämta våra data",
+    "get-data": "Använd vår data",
```

Comment:
Technically this means "Use our data" but it's better than "Hämta" which means more like Download than Get in this context. "Skaffa" could also be used but it's less formal.